### PR TITLE
#4:update style detail blog scss file

### PR DESCRIPTION
**Pull request by Lanng21**

### CODE REVIEW

**Comment by haunguyen15-tt** [comment]
No new line at the end in all files

File: src/stylesheets/components/_like-button.scss [hunk 1]
```diff
@@ -0,0 +1,14 @@
+@use "../abstracts/" as *;
+
+.like__button {
+    padding: 10px 15px;
+    background: #0080ff;
+    font-size: 18px;
+    font-family: $primary-font;
+    border: none;
+    outline: none;
+    color: #e8efff;
```
Comment (haunguyen15-tt) [inline src/stylesheets/components/_like-button.scss]: variable

File: src/stylesheets/components/_blog-detail-comment.scss [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+        &__fit-image {
+            width: 75%;
+            object-fit: cover;
+        }
+    }
+    &-content {
+        width: calc(100% - 110px);
+        &__title {
+            font-size: 16px;
+            line-height: 1;
```
Comment (baonguyen26) [inline src/stylesheets/components/_blog-detail-comment.scss]: move to common

File: src/stylesheets/components/_detail.scss [hunk 3]
```diff
@@ -0,0 +1,96 @@
+@use "../abstracts/" as *;
+
+.blog-details {
+  &-content {
+    &__image {
+      margin-bottom: 30px;
+      &-fitimg {
```
Comment (baonguyen26) [inline src/stylesheets/components/_detail.scss:7]: `img { width: '100%, height: '100%}`

**Pull request merged**